### DISCUSSION TITLE
REQUEST: Levelling checkbox in Tune Menu

### DISCUSSION THREAD

**Discussion by hippocratical** [Jyers/Marlin#1110] [Ideas]
Currently when a print is printing, in the TUNE menu there are lots of useful settings shown, like "Filament Sensor [ ]" which has a tick box to show that it is enabled.  

My printer is, shall we say, 'special' and sometimes randomly doesn't load the levelling mesh. If there was a entry in the TUNE menu that said "Levelling enabled [ ]" or even "Levelling Mesh #1 loaded", this would awesome.

**Comment by smooreace** [comment]
You would know its enabled if you load the mesh in your start GCode

**Reply by hippocratical** [reply]
You'd think that, but there's been a few times now when it hasn't worked. Sometimes on my Ender 3 the mesh gets randomly wiped  or not enabled dispite being the GCode. I'm just looking for a visual confirmation in the Tune menu of whats about to happen.

**Comment by smooreace** [comment]
Ive got somewhere around $1800 in filament through mine at this point. I've
never seen a randomly wiped EEPROM! I do update my firmware fairly
regularly though. So maybe intentionally wiping it spares me those random
"oh shit" moments
